bitex/interface/bittrex.py

```diff
@@ -103,13 +102,12 @@ def cancel_order(self, *order_ids, **kwargs):
         return results if len(results) > 1 else results[0]
 
     @format_with(BittrexFormattedResponse)
-    def wallet(self, *args, currency=None, **kwargs):  # pylint: disable=arguments-differ
+    def wallet(self, currency=None, **kwargs):
         """Return the account wallet."""
+        payload = kwargs
         if currency:
             payload = {'currency': currency}
             payload.update(kwargs)
```

Comment:
Good attempt at fixing this, but better would be:

```
payload = kwargs
if currency:
    payload.update({'currency': currency})
```

Reply:
Even better I think it's to update kwargs and pass kwargs to the params instead of a new variable called payload. I updated the entire file accordingly.